clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -811,6 +826,16 @@ private void handleFilesetCommand() {
     }
   }
 
+  private boolean hasEntity(FullName name) {
```

Comment:
The logic here is not correct as for instance table name can be null if you are dealing with a schema.